app/views/api/v1/events/_event.json.jbuilder

```diff
@@ -1,5 +1,6 @@
 json.extract! event, :id, :title, :description, :event_date, :private, :owner_id
 json.event_time event.event_time.to_formatted_s(:time)
+json.owner User.find(event.owner_id).nick
```

Comment:
`event.owner.nick` would be better, even better would be to eager load it beforehand